addons/binding/org.openhab.binding.pioneeravr/src/main/java/org/openhab/binding/pioneeravr/internal/protocol/Response.java

```diff
@@ -30,10 +30,10 @@
      *
      */
     public enum ResponseType implements AvrResponse.RepsonseType {
```

Comment:
It would be nice if you can also fix this typo: `RepsonseType` -> `ResponseType` :-)